client/python/lib/test/test_tensors.py

```diff
@@ -104,3 +104,175 @@ def test_make_ndarray_invalid_no_type():
+    assert tensor_proto.dtype == DataType.DT_INT8
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=4), TensorShapeProto.Dim(size=2)])
+
+def test_make_tensor_proto_valid_float():
+    values = [1.0, 2.0, 3.0]
+    tensor_proto = make_tensor_proto(shape=[3], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=3)])
+
+def test_make_tensor_proto_valid_scalar():
+    values = 5.0
+    tensor_proto = make_tensor_proto(shape= [], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [])
+
+def test_make_tensor_proto_valid_string():
+    values = bytes([0x13, 0x00, 0x00, 0x00, 0x08, 0x00])
+    tensor_proto = make_tensor_proto(shape=[1], dtype = DataType.DT_STRING, values = [values])
+    np_values = np.array(values)
+    assert tensor_proto.string_val == [np_values.tobytes()]
+    assert tensor_proto.dtype == DataType.DT_STRING
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=1)])
+
+def test_make_tensor_proto_valid_string_batch_size_2():
+    values = bytes([0x13, 0x00, 0x00, 0x00, 0x08, 0x00])
+    tensor_proto = make_tensor_proto(shape=None, dtype = DataType.DT_STRING, values = [values, values])
+    np_values = np.array(values)
+    assert tensor_proto.string_val == [np_values.tobytes(), np_values.tobytes()]
+    assert tensor_proto.dtype == DataType.DT_STRING
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=2)])
+
+def test_make_tensor_proto_valid_2_dims_shape():
+    values = [[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]
+    tensor_proto = make_tensor_proto(shape=[2,3], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=2), TensorShapeProto.Dim(size=3)])
+    
+def test_make_tensor_proto_valid_2_dims_shape_reshape():
+    values = [[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]
+    tensor_proto = make_tensor_proto(shape=[2,3], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+    assert tensor_proto.tensor_shape == TensorShapeProto(dim = [TensorShapeProto.Dim(size=2), TensorShapeProto.Dim(size=3)])
+
+def test_make_tensor_proto_valid_ndarray():
```

Comment:
What are we testing in this case? 
We should test either `make_tensor_proto` or `make_ndarray`. 
If one is subject to testing then the other one should be mocked and return fixed value.

If both are tested then change the name of this test to include `make_ndarray`